api/app/utils/errors.py

```diff
@@ -19,6 +19,7 @@
     {'message': 'No question exists with that Event ID'}, 404)
 FORM_NOT_FOUND = ({'message': 'No form exists with that Event ID'}, 404)
 FORM_NOT_FOUND_BY_ID = ({'message': 'No application form exists with that Application Form ID'}, 404)
+APPLICATION_FORM_EXISTS = ({'message': 'An application form exists with that Application Form ID'}, 403)
```

Comment:
Probably should say an An application form already exists for that **event** id :)